rpcs3/Emu/VFS.cpp

```diff
@@ -929,9 +929,20 @@ std::string vfs::host::hash_path(const std::string& path, const std::string& dev
 
 bool vfs::host::rename(const std::string& from, const std::string& to, const lv2_fs_mount_point* mp, bool overwrite, bool lock)
 {
+	if (fs::rename(from, to, overwrite))
```

Comment:
Not sure this is safe to do...

Reply:
The workaround that closes and reopends file handles was added for Windows because sometimes it won't allow to move a directory with opened files in it. If it succeeds once, it should be safe.

Reply:
I moved it to lock scope.